File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        unordered_map<int,int> a;
        for (int i=0; i< nums.size(); i++)
        {
            if (a.contains(nums[i])){
                a[nums[i]]=a[nums[i]]+1;
            }
            else{
                a[nums[i]]=1;
            }
            if(a[nums[i]]>1){
                return true;
            }
        }
        return false;
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def sort_by_freq(self, items: tuple):
        return -items[1]
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        dict=defaultdict(int)
        for i in range(len(nums)):
            dict[nums[i]]+=1;
        arr=sorted(dict.items(), key=self.sort_by_freq)
        ans=[]
        for i in range(k):
            ans.append(arr[i][0])
        return ans
